src/tallies/filter_mesh.cpp

```diff
@@ -50,13 +66,21 @@ MeshFilter::to_statepoint(hid_t filter_group) const
 {
   Filter::to_statepoint(filter_group);
   write_dataset(filter_group, "bins", model::meshes[mesh_]->id_);
+  if (translated_) {
+    write_dataset(filter_group, "translation", translation_);
+  }
 }
 
 std::string
 MeshFilter::text_label(int bin) const
 {
   auto& mesh = *model::meshes.at(mesh_);
-  return mesh.bin_label(bin);
+  std::string label = mesh.bin_label(bin);
+  if (translated_) {
+    label += fmt::format("\nTranslation: {}, {}, {}\n",
+                         translation_.x, translation_.y, translation_.z);
+  }
```

Comment:
I probably wouldn't bother adding this to the label. If you look at how it appears in tallies.out, it adds the translation information to every single bin which makes it a bit busy.

Reply:
Good point!
 I'll remove this.